tests/integration/test_bacnet_ingress.py

```diff
@@ -0,0 +1,39 @@
+import shlex
+import subprocess
+from pathlib import Path
+
+import pytest
+from rdflib import Namespace
+
+from buildingmotif.dataclasses import Model
+from buildingmotif.ingresses import BACnetNetwork, BACnetToBrickIngress
+from buildingmotif.namespaces import BACNET, BRICK, RDF
+
+# path to docker compose file
+docker_compose_path = Path(__file__).parent / Path("fixtures") / Path("bacnet-network")
+# command to start docker compose
+docker_compose_start = shlex.split("docker compose up -d")
+# command to stop docker compose
+docker_compose_stop = shlex.split("docker compose down")
+
+
+@pytest.mark.integration
+def test_bacnet_ingress(bm):
```

Comment:
I'd recommend moving the subprocesses to a setup/teardown fixture. https://stackoverflow.com/questions/26405380/how-do-i-correctly-setup-and-teardown-for-my-pytest-class-with-tests

Reply:
Done!